packages/react/src/component-tests/IcPopoverMenu/IcPopoverMenu.cy.tsx

```diff
@@ -251,6 +251,29 @@ describe("IcPopoverMenu end-to-end, visual regression and a11y tests", () => {
     cy.get("@icPopoverClosed").should(HAVE_BEEN_CALLED_ONCE);
   });
 
+  it("should correctly reopen when closed from a submenu", () => {
+    mount(<PopoverDropdown />);
+
+    cy.checkHydrated(POPOVER_SELECTOR);
+    // cy.checkA11yWithWait()
+
+    cy.get(POPOVER_SELECTOR).invoke(
+      "on",
+      "icPopoverClosed",
+      cy.stub().as("icPopoverClosed")
+    );
+
+    cy.get(BUTTON_SELECTOR).click();
+    cy.get("#submenu-trigger-actions").click().wait(250);
+    cy.get("body").click("bottomRight");
+    cy.get(BUTTON_SELECTOR).click();
```

Comment:
Add `cy.get("@icPopoverClosed").should(HAVE_BEEN_CALLED_ONCE);` after menu close.